Automattic/wp-calypso
# Pro Plan: add included features to plan definition

Some feature gating (e.g. "Google My Business") is handled via `planHasFeatures`. This PR adds the `getIncludedFeatures` object from the Business plan.

See: c/1KciuyFW-tr

**To test:**
- on a site with a Pro plan
- visit `/plans/my-plan`
- scroll down and click "Connect to Google My Business"
- verify that the Google My Business screen is shown
